src/components/ConversationSettings/LinkShareSettings.vue

```diff
@@ -243,7 +258,10 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.icon {
+/* FIXME: fix upstream by adding support for MD icons in buttons */
```

Comment:
What upstream? this is a plain html `button` ?

Reply:
ow, indeed... maybe the global platform CSS files about buttons ?

I think @marcoambrosini wanted to work on a generic button component so it would fit there too